packs/ossec-rootkit.conf

```diff
@@ -407,6 +407,13 @@
       "description": "tuxkit_rootkit", 
       "value": "Artifacts used by this malware", 
       "platform": "linux"
+    },
+    "reptile_rootkit": {
+      "query": "select * from file where path in ('/reptile/reptile_cmd ', '/lib/udev/reptile');", 
```

Comment:
It looks like there's an extra space in `'/reptile/reptile_cmd '`, is this intentional?

Reply:
Unintentional. It should be fixed now. Thanks for catching it!